docs-gen/content/introduction/quickstart.md

```diff
@@ -8,6 +8,10 @@ A serialization of the data definition of the vehicle signal specification is ch
 as ```vss_$VERSION.json```, where ```$VERSION``` is the content of
```

Comment:
Strictly speaking this is not entirely correct. As of today we do not always change version before tagging a release. For example, for our v2.1 tag the content of VERSION file is `2.1-develop`. In our `Makefile` we always generate files as `vss_rel_VERSION` e.g. `vss_rel_2.2-develop.json` for latest master. 

Should we possibly change to:

A serialization of the data definition of the vehicle signal specification is checked in
as ```vss_release_$VERSION.json```, where ```$VERSION``` comes from
the ```VERSION``` file.

Reply:
> Strictly speaking this is not entirely correct. As of today we do not always change version before tagging a release. For example, for our v2.1 tag the content of VERSION file is 2.1-develop. In our Makefile we always generate files as vss_rel_VERSION e.g. vss_rel_2.2-develop.json for latest master.

I'd rather see it as a bug instead of changing the docs.